walkthroughs/week-1-web-development/portfolio-walkthrough.md

```diff
@@ -127,7 +127,7 @@ cache-clearing refresh.
 
 ## Example
 
-The `examples/stanley` directory contains an example webpage. It contains
+The `walkthroughs/examples/stanley` directory contains an example webpage. It contains
```

Comment:
Do you mind changing this to be the full path? `walkthroughs/week-1-web-development/examples/stanley`

Reply:
(Sorry I just realized you closed this, feel free to ignore me.)